src/System.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -58,12 +58,9 @@ public class AsymmetricSignatureProvider : SignatureProvider
         /// </summary>
         public static readonly Dictionary<string, int> DefaultMinimumAsymmetricKeySizeInBitsForSigningMap = new Dictionary<string, int>()
         {
-            { SecurityAlgorithms.ECDSA_SHA256, 256 },
-            { SecurityAlgorithms.ECDSA_SHA384, 256 },
-            { SecurityAlgorithms.ECDSA_SHA512, 256 },
-            { SecurityAlgorithms.RSA_SHA256, 2048 },
-            { SecurityAlgorithms.RSA_SHA384, 2048 },
-            { SecurityAlgorithms.RSA_SHA512, 2048 },
+            { SecurityAlgorithms.EcdsaSha256Signature, 256 },
```

Comment:
I am concerned about reduction here. If ValidateAsymmetricSecurityKeySize can't find the algorithm, we allow it through.